api/app/controllers/spree/api/v2/platform/taxons_controller.rb

```diff
@@ -7,6 +7,14 @@ class TaxonsController < ResourceController
 
           private
 
+          def successful_reposition_actions
```

Comment:
While working on the new taxon tree using the Platform API reposition action it became apparent we need to do a little more than just update the position on a successful reposition, namely on the taxon we need to rebuild the permalink path.